pkg/storage/kubernetes/client.go

```diff
@@ -108,28 +107,31 @@ func (i *Client) ListPods(ctx context.Context) ([]v1.Pod, error) {
 }
 
 func (i *Client) GetPod(namespace, name string) (*v1.Pod, error) {
-	pod, err := i.clientSet.CoreV1().Pods(namespace).Get(context.TODO(), name, metav1.GetOptions{})
+	ctxWithTimeout, cancel := context.WithTimeout(context.Background(), storage.RequestTimeout)
```

Comment:
shall we also replace `storage.RequestTimeout` with `listRequestTimeout`?

Reply:
no, this one is 10s for a single request